### PR TITLE
Provider for k8s 1.15.1

### PR DESCRIPTION
**Pull request by rmohr**
Needed to finally have the webhook for eviction-interception.

### CODE REVIEW

File: cluster-provision/k8s/scripts/provision.sh [hunk 1]
```diff
@@ -61,7 +61,12 @@ yum install --nogpgcheck -y \
 
 # Latest docker on CentOS uses systemd for cgroup management
 # kubeadm 1.11 uses a new config method for the kubelet
```
Comment (SchSeba) [inline cluster-provision/k8s/scripts/provision.sh]: nit: can we remove this comment here?

File: cluster-provision/k8s/scripts/provision.sh [hunk 2]
```diff
@@ -145,7 +150,78 @@ $reset_command
 mkdir /etc/kubernetes/audit
 
 # New configuration for kubernetes >= 1.12
```
Comment (SchSeba) [inline cluster-provision/k8s/scripts/provision.sh]: nit: can you update this comment or remove it?

File: cluster-provision/k8s/scripts/provision.sh (hunk 1 above)
Comment (rmohr) [reply cluster-provision/k8s/scripts/provision.sh]: Done

File: cluster-provision/k8s/scripts/provision.sh (hunk 2 above)
Comment (rmohr) [reply cluster-provision/k8s/scripts/provision.sh]: Done

File: cluster-provision/k8s/scripts/provision.sh [hunk 3]
```diff
@@ -144,8 +147,78 @@ $reset_command
 # audit log configuration
 mkdir /etc/kubernetes/audit
 
-# New configuration for kubernetes >= 1.12
-if [[ ${BASH_REMATCH[1]} -ge "12" ]]; then
+if [[ ${BASH_REMATCH[1]} -ge "15" ]]; then
+cat > /etc/kubernetes/audit/adv-audit.yaml <<EOF
```
Comment (slintes) [inline cluster-provision/k8s/scripts/provision.sh]: if I see correct, we have the same audit config file 3 times now. Can move that into a single `if [[ ${BASH_REMATCH[1]} -ge "11" ]]; then ... fi` block please?

File: cluster-up/cluster/k8s-1.15.1/provider.sh [hunk 4]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+
+image="k8s-1.15.1:latest"
```
Comment (slintes) [inline cluster-up/cluster/k8s-1.15.1/provider.sh]: to probably want to update this to a shasum :slightly_smiling_face:

Comment (rmohr) [reply cluster-up/cluster/k8s-1.15.1/provider.sh]: yep, still need to test if it works properly first.

**Comment by rmohr** [comment]
/hold cancel

@slintes cluster is working now and shasum is updated.

**Pull request merged**